x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/20_search_application_put.yml

```diff
@@ -159,6 +165,18 @@ teardown:
               properties:
                 another_query_string:
                   type: "string"
+  - do:
```

Comment:
I think if we're calling get alias, and we're seeing that we have the 2 indices in the alias, we no longer need to check that the alias exists? (this applies throughout the tests)